commands/get-docker-tag/get_docker_tag.go

```diff
@@ -0,0 +1,124 @@
+package getdockertag
+
+import (
+	"bytes"
+	"fmt"
+	"github.com/go-git/go-git/v5"
+	"github.com/go-git/go-git/v5/plumbing"
+	"github.com/go-git/go-git/v5/plumbing/storer"
+	"github.com/kurtosis-tech/stacktrace"
+	"github.com/sirupsen/logrus"
+	"github.com/spf13/cobra"
+	"os"
+	"path"
+	"sort"
+	"strings"
+)
+
+const (
+	gitDirname         = ".git"
+	masterBranchName   = "master"
+	dirtySuffix        = "-dirty"
+	getDockerTagCmdStr = "get-docker-tag"
+)
+
+var GetDockerTagCmd = &cobra.Command{
+	Use:   getDockerTagCmdStr,
+	Short: "Prints the expected docker tag given the current state of the Kurtosis repo this command is run on.",
```

Comment:
This would be the `Long` description; Short needs to be really quite short due to how it's printed (e.g. `Get Docker tag of repo`).